test/data/ansible/check_ostree.yaml

```diff
@@ -48,6 +48,8 @@
 
     # case: check /boot/device-credentials exists
     # simplified installer installed Edge system ONLY
+    # This case is not stable because sometimes onboarding happens too fast and /boot/device-credentials will be deleted very soon,
```

Comment:
Could you please find a better solution to make it stable? Thanks.

Reply:
Now I will wait and check /etc/device-credentials, this file only exists after onboarding completed.